src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,69 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("size()를 통해 Set의 크기를 확인할 수 있다")
+	void size() {
+		// Act
+		int size = numbers.size();
+
+		// Assert
+		assertThat(size).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
위와 같이 한 라인으로 구현하는 것은 어떨까?